src/components/Accordion/index.js

```diff
@@ -1,6 +1,11 @@
 import React, { Component } from 'react';
```

Comment:
there are few issues:
* when the first item is focused and press arrow up key.
* when the first item is focused and press arrow down key (need to press two times).
* sometimes when the first item is focused and press arrow down key focus the last item.
*when combining tab with key it mess up.